ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/dot1x.yml

```diff
@@ -24,3 +24,20 @@ dot1x:
         apply_alternate: true
       recovery_action_reauthenticate: true
     accounting_update_interval: 6
+  captive_portal:
+    url: http://portal-nacm08/captiveredirect/
+    ssl_profile: Profile1
+    access_list_ipv4: ACL
+  supplicant:
+    profiles:
+      - name: Profile1
+        eap_method: tls
+        identity: user_id1
+        passphrase_type: 0
+        passphrase: 1048080A02
+        ssl_profile: PF1
+      - name: Profile2
+        identity: user_id2
+        passphrase: 1048080A01
```

Comment:
i did not checked then other places where we have same things defined but i think without passphrase_type it should accept unencrypted passphrase. But here we are using default as 7.
May be lets discuss on it